lldb/test/Shell/Core/x86-disabled-target.test

```diff
@@ -0,0 +1,10 @@
+# UNSUPPORTED: x86
```

Comment:
This should be:
```
UNSUPPORTED: target-x86_64
```
Which I'm pretty sure is lldb's equivalent of `x86-registered-target`. Which is whether the lldb has that target built into it, vs. whether it's running on x86.

You see a lot of requires: native && target-x86-64 for this reason.

Reply:
Actually, I'm wrong.

We get the configured targets from llvm-config and I think that adds "X86" as a feature, target- is about what the default triple of llvm is.

Reply:
Yes, I've checked that with `X86` in `LLVM_TARGETS_TO_BUILD`, the test becomes unsupported. So, it looks like it's currently OK